src/jit/gentree.h

```diff
@@ -349,6 +349,8 @@ struct GenTree
 
 #endif // FEATURE_ANYCSE
 
+    unsigned char     gtLIRFlags; // Used for nodes that are in LIR.
+
 #if ASSERTION_PROP
```

Comment:
The legal set of flags is defined in lir.h, but a pointer to the relevant type would certainly be useful.